__test__/integration/integration.test.ts

```diff
@@ -21,11 +21,18 @@ const setup = async () => {
   const extensionPath = path.join(__dirname, '../../dist');
 
   browser = await puppeteer.launch({
-    headless: false,
+    headless: true,
```

Comment:
can you make this configurable via ENV variable? So it still works with headless false when I run it locally